benchmarks/distributed/rpc/parameter_server/experiment_scripts/ddp_nccl_allreduce_sbatch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
shall we remove slurm related scripts? If you would like to keep this somewhere, is it OK to create a repo in your own github space? It's a little weird to have SLURM commands in PT source code

Reply:
okay.